s_vae/models/backbone/utils.py

```diff
@@ -1,10 +1,31 @@
-from torch import nn
+import torch
 
-
-class Reshape(nn.Module):
+class Reshape(torch.nn.Module):
     def __init__(self, shape):
         super().__init__()
         self.shape = shape
 
     def forward(self, x):
-        return x.view(-1, *self.shape)
\ No newline at end of file
+        return x.view(-1, *self.shape)
+
+
+def sample_hypersphere(n, dim =3, R = 1, seed = 0):
```

Comment:
I would not put this under backbone as this will not be used by the backbones. Not sure if we will use it for  synthetic data generation, we can move it later as well (as you mentioned). We can create a utils file in `s_vae`?